src/app/components/PlayersBoard.ts

```diff
@@ -0,0 +1,102 @@
+import { Render } from '../utils/Render';
+import { flatten } from 'lodash';
+import { Player } from '../../Player';
+
+export class PlayersBoard {
+  /* returns player's board with player's name, avatar and herd */
+  renderPlayersBoard(
+    playersChosenName: string,
+    playersChosenAvatarPath: string,
+    //(when will the color in Player) playersColor: string,
+  ): HTMLElement {
+    const player = new Player(
+      playersChosenName,
+      playersChosenAvatarPath,
+    );
```

Comment:
Player should not be created here. It is created in the game object, and passed as an argument (see comment to lines 8-9)